tests/unit/test_cli/test_core_commands.py

```diff
@@ -244,7 +244,7 @@ def test_run_command_pdb(
     app_path = f"{path.stem}:create_app"
 
     with pytest.warns(LitestarWarning):
-        result = runner.invoke(cli_command, ["--app", app_path, "run", "--pdb"])
+        result = runner.invoke(cli_command, ["--app", app_path, "run", "--use_pdb"])
```

Comment:
```suggestion
        result = runner.invoke(cli_command, ["--app", app_path, "run", "--use-pdb"])
```
We use `-` in most of the other places.